wait.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"sync"
+	"time"
+)
+
+// Wait for hosts to become ready
+func Wait(hosts []string, timeout int) {
```

Comment:
suggest using time.Duration instead of int here. clearer to have the caller to `Wait(matrix.Clients, 30 * time.Second)`